app/tests/spec/lib/key-codes.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  module.exports = {
```

Comment:
You don't really need any key-codes.js tests, there is no code there. Can you remove this file please?